tools/gfx/cpu/cpu-helper-functions.cpp

```diff
@@ -0,0 +1,18 @@
+// cpu-helper-functions.cpp
```

Comment:
This file can just be in cpu-device.cpp

Reply:
I'll merge this now and include these changes with the debug-layer split.